packages/binding-coap/package.json

```diff
@@ -25,11 +25,12 @@
     "typescript-standard": "0.3.36"
   },
   "dependencies": {
-    "wot-typescript-definitions": "0.7.3",
     "@node-wot/core": "0.7.2",
-    "rxjs": "5.5.11",
     "coap": "0.23.1",
-    "node-coap-client": "1.0.2"
+    "node-coap-client": "1.0.2",
+    "rxjs": "5.5.11",
+    "slugify": "^1.4.5",
+    "wot-typescript-definitions": "0.7.3"
```

Comment:
Please keep the original entries as is....

Reply:
Sorry, I just did `npm i slugify`, and apparently npm automatically sorted the dependencies.

Reply:
yeah, that's quite annoying... It happened also to me when I added vm2. I wonder if we can mitigate this automatic behavior by simply asking npm to order the dependencies and make a dedicated PR.

Reply:
Yes, @relu91 run into the same issue and automatic sorting makes sense. The reason why I am asking is that such changes make it difficult to merge future updates that moved on to another version and cause conflicts ...

Reply:
IMHO we could also sort the dependencies once and then just add to the contribution instructions to keep the alphabetical order in case anyone adds a new dependency.